dev_tools/utils.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 The Cirq Developers
```

Comment:
I think people get mad when you name a module utils? Maybe `test_utils`. Also this copyright date is very old :)

Reply:
renamed to test_utils!